css/datagrid/datagrid2.css

```diff
@@ -126,6 +133,8 @@
     border-right: 1px solid #00ff00;
     white-space: nowrap;
     padding-left: 4px;
+    text-overflow: ellipsis;
```

Comment:
why do we need it here?

Reply:
Just a matter of styling, will probably be changed during the final styling of the DataGrid.